src/standalone/Vegetation/canopy_energy.jl

```diff
@@ -87,11 +103,11 @@ where the canopy temperature is modeled prognostically.
 canopy_temperature(model::BigLeafEnergyModel, canopy, Y, p, t) =
     Y.canopy.energy.T
 
-function make_compute_exp_tendency(
+function make_compute_imp_tendency(
```

Comment:
do we still need a function `compute_exp_tendency!` that sets `dY.canopy.energy.T .= 0`? In our other ImEx models, we set this to avoid updating variables twice

Reply:
e.g. here in EnergyHydrology https://github.com/CliMA/ClimaLand.jl/pull/678/files#diff-7ba5ae758c389ffc507c642329638590c755bbd4a2edf53294e4dc357bf913c6R193

Reply:
ah, I just saw the fallback method for `AbstractCanopyComponent`